project/src/com/google/daggerquery/executor/models/Query.java

```diff
@@ -35,13 +37,18 @@
  * The number of parameters is completely defined by the query's name.
  */
 public class Query {
+
+  private final static String DEPS_QUERY_NAME = "deps";
+  private final static String ALLPATHS_QUERY_NAME = "allpaths";
+
   /**
    * The key is the name of supported query and the value is the number of parameters,
    * required by query with such name.
    */
   private final static Map<String, Integer> supportedQueries = new HashMap<String, Integer>() {
     {
-      put("deps", 1);
+      put(DEPS_QUERY_NAME, 1);
+      put(ALLPATHS_QUERY_NAME, 2);
     }
   };
```

Comment:
Consider using Guava's [`ImmutableMap`](https://guava.dev/releases/23.0/api/docs/com/google/common/collect/ImmutableMap.html), which supports value initialisation with the `.of()` operator or `.builder()`.